etc/ci.sh

```diff
@@ -37,18 +31,62 @@ else
         make
         cd ../..
 
-        sh bin/gap.sh --cover coverage tst/${TEST_SUITE}.g
+        sh bin/gap.sh -q tst/extractmanuals.g
 
-        # generate coverage report
-        sh bin/gap.sh -q <<GAPInput
-            if LoadPackage("profiling") <> true then
-                Print("ERROR: could not load profiling package");
-                FORCE_QUIT_GAP(1);
-            fi;
-            OutputJsonCoverage("coverage", "coverage.json");
+        for ch in tst/testmanuals/*.tst
+        do
+            COVNAME="coverage.`basename $ch .tst`"
```

Comment:
Just to clarify this is to *remove* the tst suffix from the `chapterX.tst` filename (which are `.tst`  files) to make a filename for a coverage file which then is not a `.tst` file.